physlr/physlr.py

```diff
@@ -1726,19 +1726,32 @@ def subgraph_stats_process(u):
         return Physlr.subgraph_stats(Physlr.graph, u)
 
     def physlr_subgraphs_stats(self):
-        "Retrieve subgraphs' stats."
-        gin = self.read_graph(self.args.FILES)
-        Physlr.filter_edges(gin, self.args.m)
-        print(
-            int(timeit.default_timer() - t0),
-            "Computing statistics of the subgraphs...", file=sys.stderr)
+        "Retrieve statistics of subgraphs."
+        nodes_of_interest = []
+        if len(self.args.FILES) > 1:
+            gin = self.read_graph([self.args.FILES[0]])
+            with open(self.args.FILES[1]) as fin:
+                for line in fin:
+                    nodes_of_interest.append(line.split()[0])
```

Comment:
```suggestion
                    nodes_of_interest.append(line.strip().split()[0])
```
Just to make sure we've stripped any white space